src/lib/ui/FormField.tsx

```diff
@@ -38,7 +40,11 @@ export function FormField({
       })
     }
 
-    if (component.type === TextField) {
+    if (
```

Comment:
Not required IMO. Any unknown components are just forwarded along, unchanged.

> Does this component work if more than two children are passed in?

Yep, it's just a `map` so it can accept any number of elements. 

https://github.com/seamapi/react/assets/11449462/a8a554cb-e248-4122-bb91-c6732b5d2bb7

All the labels get the same click binding!